_query-dsl/span/span-containing.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: default
+title: Span containing
+parent: Span queries
+grand_parent: Query DSL
+nav_order: 10
+---
+
+# Span containing query
+
+The `span_containing` query finds matches where a larger text pattern (like a phrase or a set of words) contains a smaller text pattern within its boundaries. Think of it as finding a word or phrase, but only when it appears within a specific larger context.
+
+For example, you can use the `span_containing` query to perform the following searches:
+
+- Find the word "quick" but only when it appears in sentences that talk about both foxes and behavior.
```

Comment:
Nit: Sentences can't technically "talk about" something.

Reply:
Changed to "mention"